tests/general/CPDBearTest.py

```diff
@@ -50,3 +50,14 @@ def test_bad_file(self):
         result = list(self.uut.run_bear_from_section([], {}))
 
         self.assertNotEqual(result, [])
+
+    def test_unsupported_language(self):
+        self.section.update_setting(
+            key='language', new_value='unsupported_language')
+
+        self.uut = CPDBear({'file_name': 'hello world  \n'},
+                           self.section,
+                           self.queue)
+
+        list(self.uut.run_bear_from_section([], {}))
+        self.assertEqual(self.uut.message_queue.queue[0].log_level, 40)
```

Comment:
And by the way, I think you should also assert something about the error message; something with `assertIn` to make sure the error message mentions… maybe `language`, if that word is in the error then it's probably really the error we're looking for.

Otherwise you run the risk of the test accidentally passing cause of something like `ERROR: CPD crashed`.